addons/account/models/account_move.py

```diff
@@ -269,6 +279,13 @@ def _get_default_invoice_incoterm(self):
     payment_state = fields.Selection(PAYMENT_STATE_SELECTION, string="Payment Status", store=True,
         readonly=True, copy=False, tracking=True, compute='_compute_amount')
 
+    # ==== Early payment cash discount fields ====
+    early_pay_amount_after_discount = fields.Float(default=0.0, compute='_compute_early_pay_amount_after_discount', store=True)
+    early_pay_amount_granted_discount = fields.Float(default=0.0, compute='_compute_early_pay_amount_granted_discount')
```

Comment:
- it must be monetary fields
- need an help message to clearly understand the purpose of those fields
- since it's related to invoice, prefix them with `invoice`